### DISCUSSION TITLE
As a starter, how to find all of the quarto properties for _quarto.yaml?

### DISCUSSION THREAD

**Discussion by Iranb** [quarto-dev/quarto-cli#3373] [Q&A] [answered]
I want to know what I can control through "_quarto.yaml" in my project, so where can I find all of the properties?

**Comment by dragonstyle** [comment, marked as answer]
The reference is a decent place to start if you're just wanting an enumeration of what is possible:

https://quarto.org/docs/reference/projects/options.html

Note that you can also look at website / book specific references.
